### ISSUE TITLE
mixed resolution multiple displays

### ISSUE THREAD

**Issue by drmbt** [theexperiential/nodeo#2]
This may not be an issue if the intention is to stream on a single moniter rig, or if they are both the same resolution, but I found the UI to freak out a bit in perform mode with a 1080 and 4k display as my outputs, drawing parts of the interface on each screen, but not the whole thing, with no way to fix directly. Setting all of my displays to 1080 fixed it, but there are obvious setups where that won't be a tenable soluition

**Comment by dylanroscover** [comment]
Yes, this is a known issue I'm working on solving with the Monitor DAT. End goal is for nodeo to function with any display configuration; by default, the control GUI should use the first monitor (fullscreen), and subsequent monitors should either be disabled (when using NDI, for example) or display content.
